mask_detector/data_prep.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import os
+
+from pathlib import Path
+from tqdm import tqdm
```

Comment:
import ordering:

1. import builtin
2. from builtin
3. import third-party
4. from third-party

```suggestion
import os
from pathlib import Path

import pandas as pd
from tqdm import tqdm
```

Reply:
Definitely good practice, thanks!